docs/examples/sentiment_analysis/self_attentive_sentence_embedding.md

```diff
@@ -1,28 +1,58 @@
-
-1. A vector representation of multiple words in a sentence can be concatenated or weighted to obtain a vector to represent the entirety of a sentence.
-
-2. Convolution (CNN) and maximum pooling (MaxPooling) on the matrix of all the word vectors of the sentence, using the final result of these techniques to represent the sentence as a whole.
-
-3. Unrolling the sentence according to the time step of the word, inputting the vector representation of each word into a recurrent neural network (RNN), and using the output of the last time step of the RNN as the representation of the sentence.
-
-The above methods solve the problem of sentence meaning, but only to a certain extent. When concatenating is used in method one, if the word of the sentence is too long and the vector dimension of the word is slightly larger, then the vector dimension of the sentence will be particularly large, and the internal interaction between the words of the sentence can not be taken into account. The use of weighted averaging is not accurate and does not adequately express the impact of each word on sentence semantics.
-
-In the second method, many useful word meanings may be lost using CNNs and MaxPooling.
-
-In the third method, the representation selected is only the output of the last step. If a sentence is too long, the output of the last step does not accurately express the entirety of the sentence's semantics.
-
-Based on the aforementioned method, Zhouhan Lin, Minwei Feng et al. published a paper [A Structured Self-attentive Sentence Embedding](https://arxiv.org/pdf/1703.03130.pdf)[1] in 2017, proposing a novel method based on self-attention structures for sentence embedding and application to users' review classification, textual entailment and other NLP tasks. In the end, better results were obtained than the previous methods.
-
-In this tutorial, we will use [GluonNLP](https://gluon-nlp.mxnet.io/index.html) to reproduce the model structure in "A Structured Self-attentive Sentence Embedding" and apply it to [Yelp Data's review star rating data set](https://www.yelp.com/dataset/challenge) for classification.
-
-## Importing necessary packages
-
-The first step, as in every one of these tutorials, is to import the necessary packages.
-
-```{.python .input}
+# A Structured Self-attentive Sentence Embedding
+
+After the novelty of word
+embeddings to create new numerical representations of words, natural language
+processing (NLP) has still been effectively improved in many ways. Along with
+the widespread use of embedding techniques, many other methods have been
+developed to further express the semantics and meanings of sentences with words:
+1. A vector representation of multiple words in a sentence can be concatenated
```

Comment:
Would you please add a blank line before 1. to fix the indentation problem?
Thanks!